c/secp256k1_blake2b_sighash_all_dual.c

```diff
@@ -181,37 +205,25 @@ validate_secp256k1_blake2b_sighash_all(uint8_t *output_public_key_hash) {
   // Digest same group witnesses
   size_t i = 1;
   while (1) {
-    len = MAX_WITNESS_SIZE;
-    ret = ckb_load_witness(temp, &len, 0, i, CKB_SOURCE_GROUP_INPUT);
+    ret = load_and_hash_witness(&blake2b_ctx, i, CKB_SOURCE_GROUP_INPUT);
     if (ret == CKB_INDEX_OUT_OF_BOUND) {
       break;
     }
     if (ret != CKB_SUCCESS) {
       return ERROR_SYSCALL;
     }
-    if (len > MAX_WITNESS_SIZE) {
-      return ERROR_WITNESS_SIZE;
-    }
-    blake2b_update(&blake2b_ctx, (char *)&len, sizeof(uint64_t));
```

Comment:
Before the hash also covers `len`, but the new version only hashes the data.

Reply:
Done